### PR TITLE
Added except description and its line number

### PR DESCRIPTION
**Pull request by juhlee-microsoft**
It will help for debugging with those information
Make sure to get all catch blocks having description and error line

Test Result
---------------------------------------------------------------
10/16/2018 21:59:36 : [INFO ] ~~~~~~~~~~~~~~~TEST END : VERIFY-SRIOV-LSPCI~~~~~~~~~~
10/16/2018 21:59:36 : [INFO ] CURRENT - PASS    - 1
10/16/2018 21:59:36 : [INFO ] CURRENT - FAIL    - 0
10/16/2018 21:59:36 : [INFO ] CURRENT - ABORTED - 0
10/16/2018 21:59:36 : [INFO ] Checking background cleanup jobs..
10/16/2018 21:59:36 : [INFO ] Removed 0 context files.
10/16/2018 21:59:36 : [INFO ] Cycle Finished.. TC-31135
10/16/2018 21:59:36 : [INFO ]
Test Results Summary
ICA test run on 10/16/2018 21:55:12
ARM Image under test Canonical : UbuntuServer : 18.04-LTS : latest
Total Executed TestCases        1 (       1 Pass,        0 Fail,        0 Abort)
Total Execution Time(dd:hh:mm) 0:0:4
XML file: TestConfiguration

#Sr. Test Name : Test Result : Test Duration (in minutes)
----------------------------------------------------
1. VERIFY-SRIOV-LSPCI : PASS : 4.33

Logs can be found at .\TestResults\2018-16-10-14-55-02-6183

10/16/2018 21:59:36 : [INFO ] Creating 'Azure-31135-buildlogs.zip' from '.\TestResults\2018-16-10-14-55-02-6183'
10/16/2018 21:59:36 : [INFO ] C:\LISAv2\PIKC31135\Azure-31135-buildlogs.zip created successfully.
10/16/2018 21:59:36 : [INFO ] Copied : .\report\report_TC-31135.xml --> .\report\report_TC-31135-junit.xml
10/16/2018 21:59:36 : [INFO ] Analysing results..
10/16/2018 21:59:36 : [INFO ] PASS  : 1
10/16/2018 21:59:36 : [INFO ] FAIL  : 0
10/16/2018 21:59:36 : [INFO ] ABORT : 0
Copying all files back to original working directory: C:\Users\juhlee\Source\Repos\LISAv2-exec.
Cleaning up C:\LISAv2\PIKC31135
Setting workspace back to original location: C:\Users\juhlee\Source\Repos\LISAv2-exec
10/16/2018 22:00:26 : [INFO ] LISAv2 exits with code: 0

### CODE REVIEW

File: Testscripts/Linux/BVT-CORE-TIME-CLOCKSOURCE.sh [hunk 1]
```diff
@@ -91,18 +96,14 @@ GetDistro
...
+		CheckSource $DISTRO
 		UnbindCurrentSource
 		;;
 	ubuntu* )
-		CheckSource
-		UnbindCurrentSource
-		;;
-	suse* )
-		CheckSource
+		CheckSource $DISTRO
```
Comment (lizzha) [inline Testscripts/Linux/BVT-CORE-TIME-CLOCKSOURCE.sh]: suse is removed

Comment (lizzha) [inline Testscripts/Linux/BVT-CORE-TIME-CLOCKSOURCE.sh]: suse is removed

File: AutomationManager.ps1 [hunk 2]
```diff
@@ -245,6 +245,9 @@ try {
 }
 
 catch {
+	$ErrorMessage =  $_.Exception.Message
```
Comment (lizzha) [inline AutomationManager.ps1]: these three lines of code occur many times, can they be put into a common function?

**Comment by iamshital** [comment]
@juhlee-microsoft If you want to fix indents in LISAv2, better fix in one single but separate PR. 
And make another PR for try/catch fix?
This would logically separate the purpose of PRs.

**Comment by juhlee-microsoft** [comment]
> @juhlee-microsoft If you want to fix indents in LISAv2, better fix in one single but separate PR.
> And make another PR for try/catch fix?
> This would logically separate the purpose of PRs.

I change the space to tab for some of old files only. Mostly the change in PR is for adding exception line in catch block. the number of files are 36 but the changes are not many.

File: AutomationManager.ps1 (hunk 2 above)
Comment (juhlee-microsoft) [reply AutomationManager.ps1]: other catch blocks have another style of printing, and I don't think I can do with a single module Sometimes they print out another information along with exception description/line. Can we do with as-is?

File: Testscripts/Linux/BVT-CORE-TIME-CLOCKSOURCE.sh (hunk 1 above)
Comment (juhlee-microsoft) [reply Testscripts/Linux/BVT-CORE-TIME-CLOCKSOURCE.sh]: Fixed 595a87642ccf3aac0de5ffe93258a80ea398c0e8

That was unintended change. Thanks for the review.

**Comment by juhlee-microsoft** [comment]
Per request, changing three lines to a module requires all catch block modification including ones that already have. I close this PR and re-create a new one.

**Pull request closed without merging**